flair/embeddings/document.py

```diff
@@ -50,9 +51,13 @@ def __init__(
         models tend to be huge.
         :param layers: string indicating which layers to take for embedding (-1 is topmost layer)
         :param layer_mean: If True, uses a scalar mix of layers as embedding
+        :param pooling: Pooling strategy for combining token level embeddings. options are 'cls', 'max', 'mean'.
         """
         super().__init__()
 
+        if pooling not in ['cls', 'max', 'mean']:
+            raise ValueError(f"Pooling operation `{poling}` is not defined for TransformerDocumentEmbeddings")
```

Comment:
The variable `poling` does not exist ;) Should be `pooling`

Reply:
Done.